src/target/riscv/riscv-013.c

```diff
@@ -742,6 +925,13 @@ static int init_target(struct command_context *cmd_ctx,
 	info->dmi_busy_delay = 0;
 	info->ac_busy_delay = 0;
 
+	// Assume all these abstract commands are supported until we learn
+	// otherwise.
```

Comment:
I've added a TODO comment to do this right, but I don't want to spend the time on this right now.